manifests/charts/base/templates/clusterrole.yaml

```diff
@@ -154,15 +154,10 @@ rules:
   - apiGroups: ["authorization.k8s.io"]
     resources: ["subjectaccessreviews"]
     verbs: ["create"]
-{{- if or .Values.global.externalIstiod }}
+{{- if .Values.global.externalIstiod }}
+  # Needed to install CA root cert on remote clusters
   - apiGroups: [""]
     resources: ["configmaps"]
     verbs: ["create", "get", "list", "watch", "update"]
-  - apiGroups: ["admissionregistration.k8s.io"]
-    resources: ["mutatingwebhookconfigurations"]
```

Comment:
We do install the mutatingwebhookconfiguration on a (non config cluster) remote cluster, but we don't support revisions there, so these aren't needed. I'll add it back if I'm mistaken (or in the future, if/when we need it).

cc @howardjohn Is this right?